cli/src/list_users.rs

```diff
@@ -0,0 +1,55 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use clap::Args;
+use std::path::PathBuf;
+
+use libparsec::get_default_config_dir;
+
+use crate::utils::*;
+
+#[derive(Args)]
+pub struct ListUsers {
+    /// Parsec config directory
+    #[arg(short, long, default_value_os_t = get_default_config_dir())]
+    config_dir: PathBuf,
+    /// Device slughash
+    #[arg(short, long)]
+    device: Option<String>,
+    /// Skip revoked users
+    #[arg(short, long, default_value_t)]
+    skip_revoked: bool,
+    /// Result offset
+    #[arg(short, long)]
+    offset: Option<usize>,
+    /// Result limit
+    #[arg(short, long)]
+    limit: Option<usize>,
```

Comment:
I'm not sure if it's useful to provide that to the user.

What was your plan with this ?
How the user would have used those options ?

Reply:
Idk, it was present with `human_find`

Reply:
Then how the user will use those options ?

Reply:
When he needs to paginate :man_shrugging:

Reply:
Do you suggest to remove it ?

Reply:
I think it would be best to do so